test/unit/dimensions.js

```diff
@@ -447,6 +447,39 @@ QUnit.test( "setters with and without box-sizing:border-box", function( assert )
 	assert.equal( el.outerHeight( 129, true ).height(), expected + 5, "test border-box innerHeight(int, true) by roundtripping" );
 } );
 
+QUnit.test( "setting ems on an element with box-sizing: border-box (gh-1712)", function( assert ) {
+	assert.expect( 20 );
+
+	var el_bb = jQuery( "<div style='margin:5px;padding:1px;border:2px solid black;-webkit-box-sizing:border-box;box-sizing:border-box;'></div>" ).appendTo( "#nothiddendiv" ),
+		el = jQuery( "<div style='margin:5px;padding:1px;border:2px solid black;'></div>" ).appendTo( "#nothiddendiv" ),
+		// font-size of nothiddendiv is 16px
```

Comment:
I suppose. Other tests would fail if the font-size on nothiddendiv was changed, but I'm fine with setting it in the test.